src/components/settings/Notifications.vue

```diff
@@ -77,6 +112,26 @@ export default {
         },
     },
 
+    watch: {
+        // Parse, store and apply new timeout settings.
+        toastSuccessTimeoutSecs: function (newTimeout) {
```

Comment:
```suggestion
        toastSuccessTimeoutSecs(newTimeout) {
```

This seems to fit better with how it is styled in the rest of the code base and in the VueJS docs.